internal/app/api/connector.go

```diff
@@ -27,51 +27,55 @@ func handleError(w http.ResponseWriter, r *http.Request, err error) {
 	}
 }
 
-func readConfig[Config payments.ConnectorConfigObject,
-	Descriptor payments.TaskDescriptor](connectorManager *integration.ConnectorManager[Config, Descriptor],
-) http.HandlerFunc {
+func readConnectorsHandler(db *mongo.Database) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		config, err := connectorManager.ReadConfig(r.Context())
+		cursor, err := db.Collection(payments.ConnectorsCollection).
+			Find(r.Context(), map[string]any{})
 		if err != nil {
 			handleError(w, r, err)
 
 			return
 		}
+		defer cursor.Close(r.Context())
 
-		err = json.NewEncoder(w).Encode(config)
+		res := make([]payments.ConnectorBaseInfo, 0)
+		if err = cursor.All(r.Context(), &res); err != nil {
+			handleError(w, r, err)
+
+			return
+		}
+
+		err = json.NewEncoder(w).Encode(
+			sharedapi.BaseResponse[[]payments.ConnectorBaseInfo]{
+				Data: &res,
+			})
 		if err != nil {
-			panic(err)
```

Comment:
We need to discuss those kind of error.
panic is not "i will do that later", it is more "i encount an unexpected error and let the caller handle it".
This allow to catch the trace and report it and upper level (with telemetry soon).